Prox/Prox/Data/TravelTimesProvider.swift

```diff
@@ -8,6 +8,8 @@ import MapKit
 
 struct TravelTimesProvider {
 
+    static fileprivate let travelTimePadding: Double = 10 * 60
```

Comment:
Nit: what unit is this? Comment, put the unit in the name or use the type system.